<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntensityProfileClass</class>
 <widget class="QMainWindow" name="IntensityProfileClass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2062</width>
    <height>1027</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AppaOCTProdSw</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QScrollArea" name="scrollArea_2">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>80</y>
          <width>1171</width>
          <height>771</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(94, 94, 94);</string>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1169</width>
           <height>769</height>
          </rect>
         </property>
         <widget class="QWidget" name="gridLayoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>1151</width>
            <height>751</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="OCTViewerGridLayout" rowstretch="0" columnstretch="0">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </widget>
       </widget>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>531</width>
          <height>871</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <widget class="QPushButton" name="pushButton_Settings">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>410</y>
           <width>40</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>download.jpg</normaloff>download.jpg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
        <widget class="QScrollArea" name="LeftEye_scrollArea">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>100</y>
           <width>200</width>
           <height>200</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>378</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>198</width>
            <height>198</height>
           </rect>
          </property>
          <widget class="QLabel" name="label_Enface">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>200</width>
             <height>200</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>254</x>
           <y>420</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;</string>
         </property>
         <property name="text">
          <string>Speed</string>
         </property>
        </widget>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>470</y>
           <width>501</width>
           <height>391</height>
          </rect>
         </property>
         <property name="title">
          <string>GroupBox</string>
         </property>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>340</y>
            <width>121</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;</string>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>461</width>
            <height>301</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>113</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>113</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 75 16pt &quot;Times New Roman&quot;;</string>
             </property>
             <property name="text">
              <string>Start_X</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>113</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>113</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 75 16pt &quot;Times New Roman&quot;;</string>
             </property>
             <property name="text">
              <string>End_X</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="lineEdit_5">
             <property name="text">
              <string>0.00</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="label_5">
             <property name="minimumSize">
              <size>
               <width>159</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 75 16pt &quot;Times New Roman&quot;;</string>
             </property>
             <property name="text">
              <string>End_X_Middle</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="label_6">
             <property name="styleSheet">
              <string notr="true">font: 75 16pt &quot;Times New Roman&quot;;</string>
             </property>
             <property name="text">
              <string>End_X_Bottom</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="lineEdit">
             <property name="text">
              <string>7.00</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="text">
              <string>-7.00</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_7">
             <property name="minimumSize">
              <size>
               <width>159</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>113</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 75 16pt &quot;Times New Roman&quot;;</string>
             </property>
             <property name="text">
              <string>Start_X_Bottom</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLineEdit" name="lineEdit_6">
             <property name="text">
              <string>0.00</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLineEdit" name="lineEdit_3">
             <property name="text">
              <string>0.00</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>159</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>113</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 75 16pt &quot;Times New Roman&quot;;</string>
             </property>
             <property name="text">
              <string>Start_X_Middle</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEdit_4">
             <property name="text">
              <string>0.00</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="pushButton_4">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>340</y>
            <width>111</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;</string>
          </property>
          <property name="text">
           <string>Galvo Stop</string>
          </property>
         </widget>
        </widget>
        <widget class="QRadioButton" name="radioButton">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>420</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 16pt &quot;Times New Roman&quot;;</string>
         </property>
         <property name="text">
          <string>6mm</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="geometry">
          <rect>
           <x>440</x>
           <y>420</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 16pt &quot;Times New Roman&quot;;</string>
         </property>
         <property name="text">
          <string>10mm</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
        <widget class="QLineEdit" name="lineEdit_7">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>420</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>3700</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>862</y>
          <width>1181</width>
          <height>61</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_Scn">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;</string>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Radial">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;;</string>
           </property>
           <property name="text">
            <string>Radial</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Cube">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;</string>
           </property>
           <property name="text">
            <string>Cube</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;</string>
           </property>
           <property name="text">
            <string>cubeScan</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_Stop">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;</string>
           </property>
           <property name="text">
            <string>Stop</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Take_Reference">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;</string>
           </property>
           <property name="text">
            <string>Take Reference</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Export">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;</string>
           </property>
           <property name="text">
            <string>Export</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>0</y>
          <width>1741</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>200</width>
           <height>50</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="IntensityProfile.qrc">:/newPrefix/logo.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>942</y>
          <width>281</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>oct live</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="IntensityProfile.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
